rlberry/envs/gym_make.py

```diff
@@ -32,14 +33,59 @@ def gym_make(id, wrap_spaces=False, **kwargs):
     return Wrapper(env, wrap_spaces=wrap_spaces)
 
 
-def atari_make(id, scalarize=True, **kwargs):
+def atari_make(id, scalarize=None, **kwargs):
     from stable_baselines3.common.env_util import make_atari_env
     from stable_baselines3.common.vec_env import VecFrameStack
 
-    env = make_atari_env(env_id=id, **kwargs)
+    # #uncomment when rlberry will manage vectorized env
+    # if scalarize is None:
+    #     if "n_envs" in kwargs.keys() and int(kwargs["n_envs"])>1:
+    #         scalarize = False
+    #     else:
+    #         scalarize = True
+
+    scalarize = True
+
+    if "atari_wrappers_dict" in kwargs.keys():
+        atari_wrappers_dict = kwargs["atari_wrappers_dict"]
```

Comment:
You can just do `atari_wrappers_dict = kwargs.pop('atari_wrappers_dict')` for the same effect.